src/source-updater.js

```diff
@@ -139,4 +133,10 @@ export default class SourceUpdater {
       callbacks[0]();
     }
   }
+
+  dispose() {
+    if (this.sourceBuffer_ && this.mediaSource.readyState === 'open') {
+      this.sourceBuffer_.abort();
```

Comment:
No expert on this pr, but should you maybe check to see if we have `this.mediaSource`?